src/components/switch/switch.tsx

```diff
@@ -115,7 +121,7 @@ export class Switch {
 
     public render() {
         return (
-            <Host>
+            <Host invalid={this.invalid}>
```

Comment:
Why is this needed? When the consumer set the `invalid` prop it will already be set on the host component :thinking:

Reply:
Oh… I didn't know. tnx

Reply:
⚡

Reply:
Clicking the "invalid" checkbox in the switch example does not seem to do anything. 
Is it due to this? Maybe it's needed anyway? I don't see how though :smile:

Reply:
I only need to the `invalid` tag in the HTML:
```html
<limel-switch invalid></limel-switch>
```
If there is a helperText, then the helper text turns red. This will happen in [this PR](https://github.com/Lundalogik/lime-elements/pull/2591) as soon as it is merged.

However, we might also want to make the label of the Switch red. I can do that later in a follow up PR.